desktop/libs/hadoop/src/hadoop/core_site.py

```diff
@@ -153,6 +154,11 @@ def get_raz_default_endpoint():
         'bucket': match.group('bucket')
       }
 
+def get_raz_default_abfs_fs():
```

Comment:
@romainr  Is this property raz specific? Can it exist in an azure environment with raz not enabled??

Reply:
No, Azure (and HDFS has the same) in general so it can exist even if no raz https://github.com/cloudera/hue/blob/master/desktop/libs/azure/src/azure/conf.py#L86 like you did in the if above (S3 equivalent is bucket + region names)

Reply:
And in general better to keep the same names anyway, so `get_raz_default_abfs_fs` --> `get_default_fs`